### PR TITLE
kustomize: add page

### LINKED ISSUE #3359: page request: kustomize
Kustomize is a tool to easily deploy resources for Kubernetes.

https://github.com/kubernetes-sigs/kustomize

### PR DESCRIPTION
**Pull request by Ameausoone**
Closes #3359

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/kustomize.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create kustomization.yaml file with resources:
+
+`kustomize create --resources deployment.yaml,service.yaml --namespace staging`
+
+- Build a kustomization.yaml file and deploy it with kubectl:
+
+`kustomize build . | kubectl apply -f -`
+
+- Edit kustomization.yaml file to set image:
```
Comment (Aracki) [inline pages/common/kustomize.md]: I would rather use the description from the command itself: `Sets images and their new names, new tags or digests in the kustomization file` 🤔

File: pages/common/kustomize.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`kustomize edit set image busybox=alpine:3.6`
+
+- Create kustomization.yaml file based on existing resources:
+
+`kustomize create --autodetect --namesuffix acme-`
```
Comment (Aracki) [inline pages/common/kustomize.md]: I think `--namesuffix acme-` is an extra thing here. I propose we go with the basic one `kustomize create --autodetect`

File: pages/common/kustomize.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`kustomize create --resources deployment.yaml,service.yaml --namespace staging`
+
+- Build a kustomization.yaml file and deploy it with kubectl:
+
+`kustomize build . | kubectl apply -f -`
+
+- Edit kustomization.yaml file to set image:
+
+`kustomize edit set image busybox=alpine:3.6`
```
Comment (pxgamer) [inline pages/common/kustomize.md]:
```suggestion
`kustomize edit set image {{busybox=alpine:3.6}}`
```

File: pages/common/kustomize.md (hunk 1 above)
Comment (Ameausoone) [reply pages/common/kustomize.md]: You're right, to be consistent with the command, I will comment command with: 
> Set image in the kustomization file

And I will change other description with the same expression "in the kustomization file"

**Comment by Ameausoone** [comment]
Thx a lot for the review @Aracki @pxgamer and @schneiderl. 

I updated my PR, I squash commits, but I'm not sure it's the better way for review PR ?

**Comment by pxgamer** [comment]
We usually squash commits when merging the PR, so in future it's a bit easier to avoid force pushing, etc.

I'll take a look now and re-review the PR. 👍

**Pull request merged**